src/main/java/no/nav/pto/veilarbportefolje/database/BrukerRepository.java

```diff
@@ -75,6 +79,49 @@ public void oppdaterSistIndeksertElastic(Timestamp tidsstempel) {
                 .collect(toList());
     }
 
+    public List<OppfolgingsBruker> hentAlleBrukereUnderOppfolging(int fromExclusive, int toInclusive) {
+        int fetchSize = 1000;
+        db.setFetchSize(fetchSize);
+
+        List<String> fnr = hentFnrFraOppfolgingBrukerTabell(fromExclusive, toInclusive);
+
+        String sql = "SELECT * FROM"
+                + " VW_PORTEFOLJE_INFO"
+                + " WHERE FODSELSNR IN (:fnr)";
+
+        List<OppfolgingsBruker> brukere = namedParameterJdbcTemplate.query(
+                sql,
+                mapOf(Pair.of("fnr", fnr)),
+                (rs, rowNum) -> erUnderOppfolging(rs) ? mapTilOppfolgingsBruker(rs, vedtakstotteFeatureErPa()) : null
+        );
+
+        return brukere.stream().filter(Objects::nonNull).collect(toList());
+    }
+
+    public List<String> hentFnrFraOppfolgingBrukerTabell(int fromExclusive, int toInclusive) {
+        String sql = "  SELECT FODSELSNR " +
+                "  FROM (SELECT " +
+                "          BRUKER.FODSELSNR, " +
+                "          rownum rn " +
+                "        FROM ( " +
+                "               SELECT * " +
+                "               FROM OPPFOLGINGSBRUKER " +
+                "               ORDER BY FODSELSNR " +
```

Comment:
hvorfør trenger du order by fnr?

Reply:
For å sørge for at man får de samme radene når man henter ut en page